update.py

```diff
@@ -18,14 +18,26 @@
     "FreeRTOS/License",
 ]
 
+source_paths_tcp = [
+	"FreeRTOS-Plus/Source/FreeRTOS-PLUS-TCP/portable/BufferManagement",
+	"FreeRTOS-Plus/Source/FreeRTOS-PLUS-TCP/portable/Compiler/GCC",
+	"FreeRTOS-Plus/Source/FreeRTOS-PLUS-TCP/include/*.h",
+	"FreeRTOS-Plus/Source/FreeRTOS-PLUS-TCP/tools",
+	"FreeRTOS-Plus/Source/FreeRTOS-PLUS-TCP/*.c",
+	"FreeRTOS-Plus/Source/FreeRTOS-PLUS-TCP/LICENSE_INFORMATION.txt",
```

Comment:
`FreeRTOS-Plus-TCP` was the issue, but due to the case-insensitive macOS file system it works locally, but not in Travis CI.